### PR TITLE
Readme updated with basic info

### PR DESCRIPTION
**Pull request by will-ball**
Closes #15 

I've just updated the README with some basic info about the centre, project and pointed to the code of conduct and contributor guidelines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
-# Relationship between social distancing measure and impact of covid-19
+# The relationship between social distancing measure and impact of covid-19: a collaborative research project
+
+
+
+## Hi! :wave:
+
+Welcome! This repository is for an ongoing collaborative project by members of the Health Data Lab team from the [Centre for Health Data Science](https://www.abdn.ac.uk/achds/), part of the University of Aberdeen.
+
+We are conducting research on the relationship between the [Grampian Social Distancing Score](https://jessbutler.github.io/social_distancing/#social-distancing-score), which was developed by [@JessButler](https://github.com/JessButler) to identify areas that might be at greater risk of transmitting COVID-19 due to likely constraints in the ability to adhere to social distancing guidelines.
```
Comment (dblana) [inline README.md]: I would say (it's wordier but clearer I think?): We are conducting research on the relationship between the [Grampian Social Distancing Score](https://jessbutler.github.io/social_distancing/#social-distancing-score) and the impact of COVID-19. The Grampian Social Distancing Score was developed by [@JessButler](https://github.com/JessButler) to identify areas that might be at greater risk of transmitting COVID-19 due to likely constraints in the ability to adhere to social distancing guidelines.

**Review by dblana — COMMENTED** [review]
How awesome! Well done Will 👍

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
...
+
+![](https://raw.githubusercontent.com/AbdnCHDS/covid-social-distancing/readme/Social%20Distancing%20in%20Grampian.png)
+
+## Contributing
+
+We encourage all members of the Aberdeen Centre for Health Data Science to contribute to this team project 😄
```
Comment (BScheliga) [inline README.md:15]: Do you want to open the contribution also up to the other parts of the University? 

"We encourage all members of the Aberdeen Centre for Health Data Science and University of Aberdeen..."

File: README.md (hunk 1 above)
Comment (will-ball) [reply README.md]: Thanks Dimitra. That sounds reasonable to me - If I update that on my local repo (in R) and commit/push, will it automatically update this pull request?

Comment (dblana) [reply README.md]: Yes it will!

Comment (will-ball) [reply README.md]: Done!

Comment (dblana) [reply README.md]: Sorry, there's a bit missing still. "We are conducting research on the relationship between GSDS" ... and what? "and the impact of COVID-19"

**Review by BScheliga — COMMENTED** [review]
Do you want to open the contribution also up to the other parts of the University?

File: README.md (hunk 2 above)
Comment (dblana) [reply README.md:15]: So for this project at least, we are restricting contributions to members of the ACHDS Github organisation (which in practice is members of the Health Data Lab Team - ACHDS & friends 😄 ). To make things easy, we are using the shared repository model for contributions (and not the fork and pull, see [here](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-collaborative-development-models) for the discussion of collaborative development models). So I think we should leave that as it is. (Anyone can comment & create issues of course, there are no restrictions there.)

File: README.md (hunk 1 above)
Comment (will-ball) [reply README.md]: 😄 sorry, just done that too

**Review by dblana — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (will-ball) [reply README.md:15]: are we happy to resolve this?

**Pull request merged**